vips/vip-490/bscmainnet.ts

```diff
@@ -0,0 +1,202 @@
+export const MIN_VOTING_DELAY = 1;
+export const MAX_VOTING_DELAY = 201600 * 2;
+
+export const NT_VOTING_PERIOD = 28800 * 2;
+export const NT_VOTING_DELAY = 1;
+export const NT_PROPOSAL_THRESHOLD = parseUnits("300000", 18);
+
+export const FT_VOTING_PERIOD = 28800 * 2;
+export const FT_VOTING_DELAY = 1;
+export const FT_PROPOSAL_THRESHOLD = parseUnits("300000", 18);
+
+export const CT_VOTING_PERIOD = 7200 * 2;
+export const CT_VOTING_DELAY = 1;
+export const CT_PROPOSAL_THRESHOLD = parseUnits("300000", 18);
+
+export const PROPOSAL_TIMELOCKS = [
+  "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396", // NT
+  "0x555ba73dB1b006F3f2C7dB7126d6e4343aDBce02", // FT
+  "0x213c446ec11e45b15a6E29C1C1b402B8897f606d", // CT
+];
+
+export const PROPOSAL_CONFIGS = [
+  [NT_VOTING_DELAY, NT_VOTING_PERIOD, NT_PROPOSAL_THRESHOLD],
+  [FT_VOTING_DELAY, FT_VOTING_PERIOD, FT_PROPOSAL_THRESHOLD],
+  [CT_VOTING_DELAY, CT_VOTING_PERIOD, CT_PROPOSAL_THRESHOLD],
+];
+
+export interface SpeedRecord {
+  market: string;
+  symbol: string;
+  supplySideSpeed: string;
+  borrowSideSpeed: string;
+}
+
+export const vip490 = () => {
+  const meta = {
+    version: "v2",
+    title:
+      "Reduce the distribution speeds and upgrade implementations considering the update of the blockrate on BNB Chain",
+    description: `Reduce the distribution speeds and upgrade implementations considering the update of the blockrate on BNB Chain`,
+    forDescription: "Execute this proposal",
+    againstDescription: "Do not execute this proposal",
+    abstainDescription: "Indifferent to execution",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: BSCMAINNET_ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [BSCMAINNET_XVS_VAULT_PROXY, "setBlocksPerYear(uint256)", NETWORK_ADDRESSES.bscmainnet.NORMAL_TIMELOCK],
+      },
+
+      {
+        target: BSCMAINNET_XVS_VAULT_PROXY,
+        signature: "setRewardAmountPerBlockOrSecond(address,uint256)",
+        params: [BSCMAINNET_XVS, BSCMAINNET_XVS_PER_BLOCK_REWARD],
+      },
+      {
+        target: BSCMAINNET_COMPTROLLER,
+        signature: "_setVenusVAIVaultRate(uint256)",
+        params: [BSCMAINNET_VAI_VAULT_RATE_PER_BLOCK],
+      },
+      {
+        target: BSCMAINNET_PLP_PROXY,
+        signature: "setTokensDistributionSpeed(address[],uint256[])",
+        params: [
+          [BSCMAINNET_BTCB, BSCMAINNET_ETH, BSCMAINNET_USDC, BSCMAINNET_USDT],
+          [
+            BSCMAINNET_BTCB_PER_BLOCK_REWARD,
+            BSCMAINNET_ETH_PER_BLOCK_REWARD,
+            BSCMAINNET_USDC_PER_BLOCK_REWARD,
+            BSCMAINNET_USDT_PER_BLOCK_REWARD,
+          ],
+        ],
+      },
+      {
+        target: BSCMAINNET_COMPTROLLER,
+        signature: "_setVenusSpeeds(address[],uint256[],uint256[])",
+        params: [
+          [BSCMAINNET_XVS_MARKET],
+          [BSC_XVS_MARKET_SUPPLY_REWARD_PER_BLOCK],
+          [BSC_XVS_MARKET_SUPPLY_REWARD_PER_BLOCK],
```

Comment:
There are no XVS rewards for borrowers in the XVS market

```suggestion
          [0],
```

Reply:
[96c75c8](https://github.com/VenusProtocol/vips/pull/531/commits/96c75c88fe78a79a8aa55a2169a6416ff7f7b2f9)